oshdb-tool/etl/src/main/java/org/heigit/ohsome/oshdb/tool/importer/util/ZGrid.java

```diff
@@ -1,5 +1,6 @@
 package org.heigit.ohsome.oshdb.tool.importer.util;
 
+import static org.heigit.ohsome.oshdb.OSHDB.coordinateToLong;
```

Comment:
there should be an empty line here

* https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing
* https://jenkins.ohsome.org/job/oshdb/view/change-requests/job/PR-320/2/checkstyle/fileName.1640957702/source.f0ea342c-43cd-46a2-aa73-1eede0ab77f9/#4

Reply:
```suggestion
import static org.heigit.ohsome.oshdb.OSHDB.coordinateToLong;

```